docs/sources/setup-grafana/configure-security/configure-authentication/saml/index.md

```diff
@@ -8,13 +8,13 @@ aliases:
   - ../../../enterprise/saml/enable-saml/
   - ../../../enterprise/saml/set-up-saml-with-okta/
   - ../../../enterprise/saml/troubleshoot-saml/
-description: Learn how to configure SAML authentication in Grafana.
-menuTitle: Configure SAML authentication
-title: Configure SAML authentication in Grafana
+description: Learn how to configure SAML authentication in Grafana's configuration file.
+menuTitle: Configure SAML using the configuration file
```

Comment:
This is gonna be long for the TOC, let's see how it looks. I'm not sure what else to recommend here. We can think on it.

Reply:
This doesn't really answer the question, but I wonder if eventually we could rearrange SAML docs even further to have something like:
- Configure authentication
   - Configure SAML
      - SAML configuration options
      - Configure SAML through the configuration file
      - Configure SAML through the user interface

In that way we clearly split reference docs about config options and task oriented docs about how to configure SAML. Otherwise this topic is currently a mix of both and has become pretty messy.